### DISCUSSION TITLE
Problems with the performance in WPF

### DISCUSSION THREAD

**Discussion by reschen** [Live-Charts/LiveCharts2#683] [Q&A]
Hi, 
I want to use LiveCharts WPF for my Bachelorthesis to display Livedata.
I testeted the Demo Version for WPF but the performance was not good. The Graph does only refresh if you change the window size.
I tried it also in WinForms, here was the performance good.
Do you have any ideas why it works so badly in WPF?

Thank you.

**Comment by beto-rodriguez** [comment]
Strange, the library shares a lot of code around all the platforms, what is the difference on what you are doing on WPF vs WinForms?

**Reply by reschen** [reply]
The problem is solved, my graphic tribers requierd updating! Now WPF is working as well as WinForms.
